### PR TITLE
Maxf/404 page

### PR DESCRIPTION
**Pull request by maxf**
Adding a gov.uk-themed 404 page. Text TBC

### CODE REVIEW

File: request_a_govuk_domain/request/templates/404.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      <h1 class="govuk-heading-xl">
+        Page not found
+      </h1>
+
+      <p class="govuk-body">
+        If you entered a web address, check it is correct.
+      </p>
+
+      <p class="govuk-body">
+        You can <a class="govuk-link" href="/">go back to the service's start page</a>.
```
Comment (johnalupton-dco) [inline request_a_govuk_domain/request/templates/404.html:19]: maybe "home page"? no strong opinion

File: start-server.sh [hunk 2]
```diff
@@ -1,3 +1,4 @@
 #!/bin/sh
-echo "Starting test server"
+echo "Starting test server with DEBUG set to False"
```
Comment (johnalupton-dco) [inline start-server.sh:2]: Is this always and only for debug? 

Maybe switch the order and have `echo "Starting test server with DEBUG set to ${DEBUG}"` ?

File: request_a_govuk_domain/request/views.py [hunk 3]
```diff
@@ -621,3 +621,7 @@ def service_failure_view(request):
     )
```
Comment (dhvander) [inline request_a_govuk_domain/request/views.py:621]: I know this is not one of the changed lines, is this a GDPR violation to dump the data that may include user information?

File: request_a_govuk_domain/request/templates/404.html (hunk 1 above)
Comment (maxf) [reply request_a_govuk_domain/request/templates/404.html]: Changed. I think it's going to be rewritten but no strong opinion either.

**Review by dhvander — APPROVED** [review]

File: start-server.sh (hunk 2 above)
Comment (maxf) [reply start-server.sh:2]: actually I meant to remove that. Thanks.

File: request_a_govuk_domain/request/views.py (hunk 3 above)
Comment (maxf) [reply request_a_govuk_domain/request/views.py]: indeed. Removing

**Pull request merged**